.github/workflows/build.yml

```diff
@@ -1,28 +1,57 @@
 name: Compatibility Test
 
 on:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
```

Comment:
Running this one for each commit could be an overkill.
Currently it is running one something is merged into `develop`